src/VisualStudio/Core/Def/FindReferences/StreamingFindUsagesPresenter.cs

```diff
@@ -277,12 +277,12 @@ private static (Guid, string projectName, string? projectFlavor) GetGuidAndProje
         // certain features (like filtering) may not work in that context.
         var vsWorkspace = document.Project.Solution.Workspace as VisualStudioWorkspace;
 
-        var (projectName, projectFlavor) = document.Project.State.NameAndFlavor;
+        var (projectName, _) = document.Project.State.NameAndFlavor;
```

Comment:
>NameAndFlavor [](http://example.com/codeflow?start=54&length=13)

There is a ProjectState.Name, is that different?

Reply:
It is.  .Name will return `"Microsoft.CodeAnalysis.CSharp (net7.0)"` (so the name, including the flavor/tfm).  This is the helper to say "i explicitly do not want the flavor, i'm collapsing into a single shared view of all tfms"